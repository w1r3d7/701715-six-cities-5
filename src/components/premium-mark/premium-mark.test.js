import React from 'react';
import renderer from 'react-test-renderer';

import PremiumMark from './premium-mark';

const cardType = `place-card`;

describe(`PremiumMark render correctly`, () => {
  it(`It is premium`, () => {
    const tree = renderer
      .create(
          <PremiumMark
            cardType={cardType}
            isPremium={true}
          />
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`It isn't premium`, () => {
    const tree = renderer
      .create(
          <PremiumMark
            cardType={cardType}
            isPremium={false}
          />
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});

