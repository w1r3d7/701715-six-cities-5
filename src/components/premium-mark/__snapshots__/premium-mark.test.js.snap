// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PremiumMark render correctly It is premium 1`] = `
<div
  className="place-card__mark"
>
  <span>
    Premium
  </span>
</div>
`;

exports[`PremiumMark render correctly It isn't premium 1`] = `null`;
